src/Nimbus/Infrastructure/LongRunningTasks/LongRunningTaskWrapperBase.cs

```diff
@@ -111,6 +111,13 @@ private Task Watch(ILongRunningTask longRunningHandler, BrokeredMessage message)
                     return;
                 }
 
+                object dispatchAbandoned;
+                if (message.Properties.TryGetValue(MessagePropertyKeys.DispatchAbandoned, out dispatchAbandoned) && dispatchAbandoned as bool? == true)
+                {
+                    //_logger.Debug("Long-running task wrapper awoke after message {0} had already been dispatched. Nothing to see here.", message.MessageId);
```

Comment:
I kept that because the check for DispatchComplete also has a log commented out (line 110) so did it to indicate it's a similar check.